opsdroid/connector/sms/__init__.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Opsdroid Imports
+from opsdroid.connector import Connector, register_event
+from opsdroid.events import Message
+import logging
+
+# Twilio Imports
+from twilio.rest import Client
+
+_LOGGER = logging.getLogger(__name__)
+CONFIG_SCHEMA = {
+    "name": str,
+    "account_sid": str,
+    "auth_token": str,
+    "phone_number": str,
+    "host": str,
+    "port": int,
+}
+
+
+class ConnectorSMS(Connector):
+    """A connector for Twilio-powered SMS"""
+
+    def __init__(self, config, opsdroid):
+        super().__init__(config, opsdroid=opsdroid)
+        self.name = self.config.get("name", "sms")
+        self.bot_name = config.get("bot-name", "opsdroid")
+        self.config = config
+
+    async def connect(self):
+        """Connect to Twilio and setup webhooks"""
+        self.connection = await Client(
+            self.config["account_sid"], self.config["auth_token"]
+        )
+        self.opsdroid.web_server.web_app.router.add_get(
+            f"/connector/{self.name}", self.handle_messages
+        )
```

Comment:
This is the correct way to add a get endpoint to the web server. 

Do you have to manually configure Twilio to know what your opsdroid webhook URL is? In a web GUI or something?

Reply:
Yes, I couldn't find a way to do it via the REST api.